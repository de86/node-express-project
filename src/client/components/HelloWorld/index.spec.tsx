import * as React from 'react';
import { renderIntoDocument } from 'react-dom/test-utils';
import HelloWorld from '.';

describe('/components/HelloWorld', () => {
  it('should match the HelloWorld snapshot', () =>
    expect(renderIntoDocument(<HelloWorld />)).toMatchSnapshot('HelloWorld') );

});
