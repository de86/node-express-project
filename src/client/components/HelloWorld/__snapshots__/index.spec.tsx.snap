// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/components/HelloWorld should match the HelloWorld snapshot: HelloWorld 1`] = `null`;
